tensorflow_lite_support/c/task/vision/object_detector.h

```diff
@@ -0,0 +1,198 @@
+/* Copyright 2021 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+#ifndef TENSORFLOW_LITE_SUPPORT_C_TASK_VISION_IMAGE_CLASSIFIER_H_
```

Comment:
This should be `TENSORFLOW_LITE_SUPPORT_C_TASK_VISION_OBJECT_DETECTOR_H_`

Reply:
@priankakariatyml We accidentally merged this pull request internally so please fix it in a future CL.